starknet_devnet/util.py

```diff
@@ -99,7 +129,11 @@ def parse_args():
     #          "or network name (alpha or alpha-mainnet)",
     # )
 
-    return parser.parse_args()
+    args = parser.parse_args()
+    if args.dump_on and not args.dump_path:
```

Comment:
Why is this check necessary?

Reply:
I think it doesn't make sense to allow user to specify **when** to dump, and not specify **where** to dump.